data/random-teams.ts

```diff
@@ -1524,7 +1550,9 @@ export class RandomTeams {
 			ability !== 'Sturdy' &&
 			(species.baseStats.spe >= 90 || !hasMove['voltswitch']) &&
 			['foulplay', 'rapidspin', 'substitute', 'uturn'].every(m => !hasMove[m]) && (
-				counter.speedsetup || counter.drain ||
+				counter.speedsetup ||
+				// No Dyna Buzzwole doesn't want Life Orb with Bulk Up + 3 attacks
```

Comment:
You say this applies only to No Dyna Buzzwole, but never check if we're in the normal or no-Dynamax format.